### DISCUSSION TITLE
EDC Extension for AAS

### DISCUSSION THREAD

**Discussion by JensMueller2709** [eclipse-edc/Connector#2451] [Show and tell]
# Adoption Request

Get the [EDC Extension for AAS](https://github.com/FraunhoferIOSB/EDC-Extension-for-AAS) referenced as friend.

## General information

[Asset Administration Shell (AAS)](https://www.plattform-i40.de/SiteGlobals/IP/Forms/Listen/Downloads/EN/Downloads_Formular.html?cl2Categories_TechnologieAnwendungsbereich_name=Verwaltungsschale) data can be manually shared over the EDC by linking an EDC Asset to the HTTP endpoint of the specific AAS element. Additionally, contracts and policies have to be defined for each element. 

In order to minimize configuration effort and prevent errors, this extension is able to link existing AAS services and its elements to the EDC automatically. Furthermore, this extension can also start an AAS service by reading an static AAS model file. A default contract and policy can be chosen to be applied for all elements. For critical elements, additional contracts and policies can be placed. External changes to the structure of an AAS are automatically synchronized by the extension.

| Title | Description | Links
| :--- | :--- | :--- 
| EDC Extension for Asset Administration Shell (AAS) | This extension for the EDC provides an easy way to share an [Asset Administration Shell (AAS)](https://www.plattform-i40.de/SiteGlobals/IP/Forms/Listen/Downloads/EN/Downloads_Formular.html?cl2Categories_TechnologieAnwendungsbereich_name=Verwaltungsschale) model via the EDC. | [Github Repo](https://github.com/FraunhoferIOSB/EDC-Extension-for-AAS)

## Adoption level

Next, please tell us what level of adoption you intend to provide. _(pick only one)_

- [x] Reference a feature as "friend"
- [ ] Incorporate a feature as core EDC component

**Comment by juliapampus** [comment]
Hey @JensMueller2709, we added your project to our "known friends" list 🔥 We thought it may be a nice idea to give EDC adopters the opportunity to present their projects to the team and the community. We would use our Q&A session for this. You can find an overview of the next scheduled meetings in #1303.

It would be nice if you could let me know whether you like the idea and, if so, which date fits.
